pkg/driver/node.go

```diff
@@ -227,6 +230,38 @@ func (d *nodeService) NodeUnstageVolume(ctx context.Context, req *csi.NodeUnstag
 	return &csi.NodeUnstageVolumeResponse{}, nil
 }
 
+func (d *nodeService) NodeExpandVolume(ctx context.Context, req *csi.NodeExpandVolumeRequest) (*csi.NodeExpandVolumeResponse, error) {
+	klog.V(4).Infof("NodeExpandVolume: called with args %+v", *req)
+	volumeID := req.GetVolumeId()
+	if len(volumeID) == 0 {
+		return nil, status.Error(codes.InvalidArgument, "Volume ID not provided")
+	}
+
+	args := []string{"-o", "source", "--noheadings", "--target", req.GetVolumePath()}
+	output, err := d.mounter.Run("findmnt", args...)
+	if err != nil {
+		return nil, status.Errorf(codes.Internal, "Could not determine device path: %v", err)
+
+	}
+
+	devicePath := strings.TrimSpace(string(output))
+	if len(devicePath) == 0 {
+		return nil, status.Errorf(codes.Internal, "Could not get valid device for mount path: %q", req.GetVolumePath())
+	}
+
+	// TODO: refactor Mounter to expose a mount.SafeFormatAndMount object
+	r := resizefs.NewResizeFs(&mount.SafeFormatAndMount{
+		Interface: mount.New(""),
+		Exec:      mount.NewOsExec(),
+	})
```

Comment:
@leakingtapan: we might need to refactor the `Mounter` interface, but I'd like to avoid doing that in this PR.

Reply:
I think the [Mounter interface](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/pkg/driver/mount.go) already implements `mount.SafeFormatAndMount`? Could you try:
```
	r := resizefs.NewResizeFs(&d.mounter)
```

Reply:
`cannot use &d.mounter (type *Mounter) as type *mount.SafeFormatAndMount in argument to resizefs.NewResizeFs`

`NewResizeFs` won't take `d.mounter` as argument because `SafeFormatAndMount` is a struct (not an interface).